.gitlab/ci/.gitlab-ci.bucket-upload.yml

```diff
@@ -709,6 +709,13 @@ upload-id-set-bucket:
   extends:
     - .bucket-upload-rule
     - .default-job-settings
+  variables:
+    PRODUCT_TYPE: "XSOAR"
+    SERVER_TYPE: "XSOAR"
+    INSTANCE_ROLE: "Server Master"
+    ARTIFACTS_FOLDER: "${ARTIFACTS_FOLDER_XSOAR}"
+    ARTIFACTS_FOLDER_INSTANCE: "${ARTIFACTS_FOLDER}/instance_${INSTANCE_ROLE}"
+    ARTIFACTS_FOLDER_SERVER_TYPE: "${ARTIFACTS_FOLDER}/server_type_${SERVER_TYPE}"
```

Comment:
Are these the correct paths? with `instance_` and `server_type_`?

Reply:
yes, they're correct. they are computed from the previous variables.